### ISSUE TITLE
thread '<unknown>' has overflowed its stack

### ISSUE THREAD

**Issue by MicahZoltu** [paritytech/parity#6821]
> I'm running:
>
> - **Parity version**: `parity/parity:stable (28073c859259)` / `Parity/v1.7.7-stable-eb7c648-20171015/x86_64-linux-gnu/rustc1.20.0`
> - **Operating system**: Linux
> - **And installed**: Docker

---

Failure output
```
Loading config file from /parity/instant-seal-config.toml
2017-10-18 17:10:43 UTC Starting Parity/v1.7.7-stable-eb7c648-20171015/x86_64-linux-gnu/rustc1.20.0
2017-10-18 17:10:43 UTC Keys path /parity/keys/InstantSealChain
2017-10-18 17:10:43 UTC DB path /parity/chains/InstantSealChain/db/e8864f27cfc9038b
2017-10-18 17:10:43 UTC Path to dapps /parity/dapps
2017-10-18 17:10:43 UTC State DB configuration: fast
2017-10-18 17:10:43 UTC Operating mode: active
2017-10-18 17:10:43 UTC Configured for InstantSealChain using InstantSeal engine
2017-10-18 17:10:43 UTC Transaction mined (hash 6784b2f532954cf5b9f81cdbacb80b9bedbea7360bf00011bf2539055a36a909)
thread '<unknown>' has overflowed its stack
fatal runtime error: stack overflow
```
Dockerfile:
```
FROM parity/parity:stable

# install all dependencies
RUN apt-get update \
	&& apt-get install --yes --no-install-recommends curl \
	&& rm -rf /var/lib/apt/lists/*

WORKDIR /
COPY dev-key.json /parity/keys/InstantSealChain/dev-key.json
COPY instant-seal-chain-spec.json /parity/instant-seal-chain-spec.json
COPY instant-seal-config.toml /parity/instant-seal-config.toml
COPY start.sh /start.sh
RUN echo "" > /parity/password
RUN chmod +x /start.sh

ENTRYPOINT [ "/start.sh" ]
```
dev-key.json
```
{
    "id": "21c82dac-65cd-df3f-9750-f36073fcc51a",
    "version": 3,
    "crypto": {
        "cipher": "aes-128-ctr",
        "cipherparams": {
            "iv": "b821de8c19c6b14ec38353026612049f"
        },
        "ciphertext": "0e7cacb926f126344b0a4c606b2fc981fb5ef26867a297e370008dd8a8200e60",
        "kdf": "pbkdf2",
        "kdfparams": {
            "c": 10240,
            "dklen": 32,
            "prf": "hmac-sha256",
            "salt": "a8dd39be8552b82416d7af150f3d25bb1850660e63b512b5bc94d0685e31ff92"
        },
        "mac": "4a2153a0a6ac6479e30dd4259f54e047cd872190ca463cfbf1393e171ab6a26a"
    },
    "address": "913da4198e6be1d5f5e4a40d0667f70c0b5430eb",
    "name": "",
    "meta": "{}"
}
```
instant-seal-config.toml
```
[parity]
# Local Development Chain
chain = "/parity/instant-seal-chain-spec.json"
# Forces Parity to run even if there are known issues regarding consensus. Not recommended.
no_consensus = true
# Blockchain and settings will be stored in /parity.
base_path = "/parity"
# Parity databases will be stored in /parity/chains.
db_path = "/parity/chains"
# Your encrypted private keys will be stored in /parity/keys.
keys_path = "/parity/keys"

[account]
# From: [0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb] you'll be able to send tranasactions without password.
unlock = ["0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb"]
# File at /parity/password should contain passwords to unlock your accounts. One password per line.
password = ["/parity/password"]

[network]
# Enable or disable new peers discovery.
discovery = false
# Connect only to reserved nodes.
reserved_only = true

[rpc]
#  JSON-RPC will be listening for connections on IP 0.0.0.0.
interface = "all"
# Allows Cross-Origin Requests from domain '*'.
cors = "*"
# Only selected APIs will be exposed over this interface.
apis = ["all"]
# Allow connections only using specified addresses.
hosts = ["all"]

[websockets]
#  JSON-RPC will be listening for connections on IP 0.0.0.0.
interface = "all"
# Allows connecting from Origin '*'.
origins = ["all"]
# Only selected APIs will be exposed over this interface.
apis = ["all"]
# Allow connections only using specified addresses.
hosts = ["all"]

[ipc]
# You won't be able to use IPC to interact with Parity.
disable = true

[secretstore]
# You won't be able to encrypt and decrypt secrets.
disable = true

[ipfs]
# You won't be able to hash-query blockchain data.
disable = true

[dapps]
# You won't be able to access any web Dapps.
disable = true

[mining]
reseal_min_period = 0
reseal_max_period = 0
reseal_on_txs = "all"
force_sealing = false

min_gas_price = 1
# Targeting 6500000 gas per block when sealing a new block.
gas_floor_target = "6500000"
# Gas limit will be raised at most by 6500000 gas.
gas_cap = "6500000"
# Parity will reject transactions above 6500000 gas.
tx_gas_limit = "6500000"
# Parity will not save local transaction queue to disk.
no_persistent_txqueue = true

[footprint]
# Significant speed up but unclean exit is unrecoverable (disables DB WAL).
fast_and_loose = true

[snapshots]
# Disables automatic periodic snapshots.
disable_periodic = true
```
instant-seal-chain-spec.json
```
{
    "name": "InstantSealChain",
    "engine": {
        "instantSeal": { "params": {} }
    },
    "params": {
        "gasLimitBoundDivisor": "0x400",
        "accountStartNonce": "0x0",
        "maximumExtraDataSize": "0x20",
        "minGasLimit": "0x1",
        "networkID" : "0x11"
    },
    "genesis": {
        "seal": {
            "generic": "0x0"
        },
        "difficulty": "0x20000",
        "author": "0x0000000000000000000000000000000000000000",
        "timestamp": "0x00",
        "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
        "extraData": "0x",
        "gasLimit": "0x632EA0"
    },
    "accounts": {
        "0x0000000000000000000000000000000000000001": { "balance": "1", "builtin": { "name": "ecrecover", "pricing": { "linear": { "base": 3000, "word": 0 } } } },
        "0x0000000000000000000000000000000000000002": { "balance": "1", "builtin": { "name": "sha256", "pricing": { "linear": { "base": 60, "word": 12 } } } },
        "0x0000000000000000000000000000000000000003": { "balance": "1", "builtin": { "name": "ripemd160", "pricing": { "linear": { "base": 600, "word": 120 } } } },
        "0x0000000000000000000000000000000000000004": { "balance": "1", "builtin": { "name": "identity", "pricing": { "linear": { "base": 15, "word": 3 } } } },
        "0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb": { "balance": "0xffffffffffffffffffffffffffffffff"}
    }
}
```
start.sh
```
set -m
/parity/parity --config /parity/instant-seal-config.toml --gasprice 1 &
while ! curl --silent --show-error localhost:8545 -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"net_version","id": 1}'; do sleep 0.1; done
curl --silent --show-error localhost:8545 -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"eth_sendTransaction","params":[{"value":"0x0","to":"0x0000000000000000000000000000000000000000","from":"0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb","data":"0x","gasPrice":"0x1"}], "id": 1}'
fg
```
commands
```
docker image build --tag parity-dev-node .
docker container run --rm -it --name parity-dev-node parity-dev-node
```

**Comment by MicahZoltu** [comment]
Deleting the following lines appears to work around the crash:
```
reseal_min_period = 0
reseal_max_period = 0
reseal_on_txs = "all"
force_sealing = false
```

**Comment by 5chdn** [comment]
The mother of all bugs :)

**Comment by alex-miller-0** [comment]
I was having the same problem, but it appears to be fixed in 1.8.4

**Comment by debris** [comment]
@MicahZoltu please reopen if it still occurs
